### ISSUE TITLE
Tooltips aren't accessibility compliant 

### ISSUE THREAD

**Issue by paramsinghvc** [highcharts/highcharts#19558]
#### Expected behaviour
The tooltip in highcharts lack `role="tooltip"` on them and also there's no `aria-describedby` attribute applied onto the charts points themselves which makes the point and the tooltip linked to each other.
As per the spec, https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role
the `aria-descirbedby` and `role="tooltip"` must be applied for tooltips implementation.

#### Actual behaviour
There is no role="tooltip" on the tooltip elements and no aria-describedby on the series points and bars themselves to help them explain about the contents when the screen reader navigates there.

![Screenshot 2023-08-15 at 14 02 19](https://github.com/highcharts/highcharts/assets/4329912/2addb5ae-00a9-492e-a7ef-18ea85810479)

#### Live demo with steps to reproduce
https://jsfiddle.net/5s1Lzh2c/

#### Product version

v8 and v11 

#### Affected browser(s)
Chrome

**Comment by oysteinmoseng** [comment]
Hi @paramsinghvc, this behavior is by design, and the data points have `aria-label`s instead. Through user research we found that the verbosity of the tooltips is usually not suitable for BVI screen reader users, so a different format for the accessible names will often be needed. Ideally we would make use of something like `aria-details` to reach the more verbose (and potentially HTML-structured) tooltip, but support is still poor for this. 

You can format the accessible names of the data points to match the information in the tooltip by altering the chart configuration, for example using [`accessibility.point.valueDescriptionFormat`](https://api.highcharts.com/highcharts/accessibility.point.valueDescriptionFormat).

With that said, we are considering adding an option to more easily match the tooltip format for those who are looking for this.

For now, I am closing this as not a bug, but feel free to comment if you have additional information, and we can reopen.
